includes/import/class-wc-product-csv-importer.php

```diff
@@ -530,9 +530,10 @@ public function parse_images_field( $value ) {
 			return array();
 		}
 
-		$images = array();
+		$images    = array();
+		$separator = apply_filters( 'woocommerce_product_import_image_separator', ',' ); 
```

Comment:
There's an empty space at the end of this line, can you perhaps just remove that then this is good to merge.

Reply:
> There's an empty space at the end of this line

Sorry about that. Removed.